src/cromshell/cost/command.py

```diff
@@ -182,15 +182,15 @@ def create_bq_query(detailed: bool, bq_cost_table: str) -> str:
                 AND task.key LIKE "wdl-task-name"
                 AND wfid.key LIKE "cromwell-workflow-id"
                 AND wfid.value like @workflow_id
-                AND partition_time BETWEEN @start_date AND @end_date
+                AND export_time BETWEEN @start_date AND @end_date
```

Comment:
I looked at our cost tabe and do see this key in it, instead of the `partition_time` key. But I wonder why it was `partition_time` at the beginning, starting from the shell version. Did Google change things underneath us?

`export_time `is documented here, but not `partition_time`
https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage

And confusing the situation, even more, are the two other `*time` values: `usage_start_time` and `usage_end_time`. And now I wonder, should we use them instead of `export_time`?